src/test/kotlin/lotto/LottoStatisticsTest.kt

```diff
@@ -36,4 +57,63 @@ class LottoStatisticsTest {
 
         assertThat(isLoss).isFalse()
     }
+
+    @ParameterizedTest
+    @MethodSource("rankTestData")
+    fun `등수 별로 일치갯수를 알 수 있다`(
+        winningNumbers: Lotto,
+        userLottos: Lottos,
+        bonusNumber: Int,
+        expectedRank: Rank,
+    ) {
+        val user = User(Amount(1000))
```

Comment:
전 개인적으로 금액을 노출할 때에는 다음과 같이 쉼표 대신 구분해주는 것이 가독성에 좋더라고요!
```suggestion
        val user = User(Amount(1_000))
```